sdks/python/src/opik/llm_usage/opik_usage.py

```diff
@@ -35,6 +35,13 @@ class OpikUsage(pydantic.BaseModel):
     completion_tokens: Optional[int] = None
     prompt_tokens: Optional[int] = None
     total_tokens: Optional[int] = None
+    
+    # Multimodal usage tracking
+    audio_input_tokens: Optional[int] = None
```

Comment:
No changes at all are required from the SDK side in this PR, all this information is already sent to the backend.

The fields
```python
    completion_tokens: Optional[int] = None
    prompt_tokens: Optional[int] = None
    total_tokens: Optional[int] = None
```
are needed only for 2 things in Opik:
1. FE uses them to display input/output/total tokens.
2. Backend may use them as a fallback for price calculation. But it's an old logic, today backend should use the usage fields which names start with `original_usage.*`.